tools/build_defs/docker/utils.py

```diff
@@ -45,12 +47,16 @@ def GetTarFile(f, name):
   Returns:
     The content of the file, or None if not found.
   """
-  with tarfile.open(f, 'r') as tar:
-    members = [tarinfo.name for tarinfo in tar.getmembers()]
-    for i in ['', './', '/']:
-      if i + name in members:
-        return tar.extractfile(i + name).read()
-    return None
+  with open(f, "rb") as file:
```

Comment:
`file` is also a built-in type, so its use can be confusing in line 52 (it managed to confuse me). Please rename it.

Reply:
Done.